### PR TITLE
PLG-18: Magento User Groups Sync

### PR DESCRIPTION
**Pull request by nedick007**
Apply customer group as tags param during customer events.

### CODE REVIEW

**Comment by nedick007** [comment]
@adamof ready for review

File: Helper/CustomerSerializer.php [hunk 1]
```diff
@@ -5,20 +5,25 @@
...
+        \Magento\Customer\Api\GroupRepositoryInterface $groupRepository
     ) {
         $this->subscriberModel = $subscriberModel;
+        $this->groupRepository = $groupRepository;
     }
     
     public function serialize($customer) {
         $this->subscriberModel->unsetData();
+        $group       = $this->groupRepository->getById($customer->getGroupId());
+        $groupName[] = $group->getCode();
```
Comment (adamof) [inline Helper/CustomerSerializer.php]: Can the customer belong only to one group?

Comment (nedick007) [reply Helper/CustomerSerializer.php]: In Magento2 customer can belong only to one group without system modification.

Comment (adamof) [reply Helper/CustomerSerializer.php]: What if a customer doesn't belong to any group?

Comment (nedick007) [reply Helper/CustomerSerializer.php]: This is not possible. Customer group is required for every customer.

**Comment by nedick007** [comment]
Tested on QA env.

**Pull request merged**